api/envoy/config/filter/network/mongo_proxy/v2/mongo_proxy.proto

```diff
@@ -19,6 +19,7 @@ option (udpa.annotations.file_status).package_version_status = FROZEN;
 // MongoDB :ref:`configuration overview <config_network_filters_mongo_proxy>`.
 // [#extension: envoy.filters.network.mongo_proxy]
 
+// [#next-free-field: 6]
```

Comment:
v2 is locked/deprecated, so please remove these changes.

Reply:
Hm, so I originally didn't include v2 in this change but I got test failures locally.

Reply:
Specifically, a segfault in what looked to be an attempt to convert v3 into v2 config.

Reply:
Anyway, removed, hopefully CI passes.

Reply:
Looks good, I think I might have been testing this change internally on an older git commit.